apps/api/src/utils/aliases.ts

```diff
@@ -0,0 +1,196 @@
+		const accumulator = { ...acc, [aliasedKey]: value };
+		return accumulator;
+	}, {}) as DatabaseRecord;
+
+	return databaseData;
+};
+
+/** Convenience function for specific alias utils input/output scenarios */
+export const aliasApplicationRecord = (data: JoinedApplicationRecord): ApplicationResponseData => {
+	const {
+		id,
+		user_id: userId,
+		state,
+		created_at: createdAt,
+		approved_at: approvedAt,
+		updated_at: updatedAt,
+		expires_at: expiresAt,
+		contents: applicationContents,
+	} = data;
+
+	const omittedKeys = [
+		'id',
+		'applicant_signature',
+		'applicant_signed_at',
+		'institutional_rep_signature',
+		'institutional_rep_signed_at',
+		'ethics_review_required',
+		'ethics_letter',
+		'signed_pdf',
+	];
+
+	const contents = applicationContents
+		? aliasToResponseData<ApplicationContentUpdates, ApplicationContentsResponse>(applicationContents, omittedKeys)
+		: null;
+
+	return {
+		id,
+		userId,
+		state,
+		createdAt,
+		approvedAt,
+		updatedAt,
+		expiresAt,
+		contents,
+	};
+};
+
+/** Convenience function for specific alias utils input/output scenarios */
+export const aliasApplicationContentsRecord = (update: UpdateEditApplicationRequest): ApplicationContentUpdates => {
+	const omitKeys = [
+		'id',
+		'applicantSignature',
+		'applicantSignedAt',
+		'institutionalRepSignature',
+		'institutionalRepSignedAt',
+	];
+	const formattedUpdate = aliasToDatabaseData<UpdateEditApplicationRequest, ApplicationContentUpdates>(
+		update,
+		omitKeys,
+	);
+
+	return formattedUpdate;
+};
+
+/**
+ * Helper function to convert Postgres snake_case to FE camelCase for the Signature Service
+ * @param data type `ApplicationSignatureUpdate` - Signature fields + application_id from the DB
+ * @returns type `SignatureDTO` - camelCase variation of a Postgress success response.
+ */
+export const aliasSignatureRecord = (data: ApplicationSignatureUpdate): SignatureDTO => {
+	const responseData = aliasToResponseData<ApplicationSignatureUpdate, SignatureDTO>(data);
+
+	return responseData;
+};
+
+/**
+ * Helper function to change CollaboratorRecord db keys to DTO Collaborator camelCase keys
+ * @param data type CollaboratorRecord in snake_case
+ * @returns  type GetCollaboratorsResponse in camelCase w/ Collaborator added
+ */
+export const aliasCollaboratorRecord = (data: CollaboratorRecord[]): GetCollaboratorsResponse[] => {
+	const formattedUpdate: GetCollaboratorsResponse[] = [];
+
+	data.forEach((value) => {
+		formattedUpdate.push({
+			id: value.id,
+			applicationId: value.application_id,
+			collaboratorFirstName: value.first_name,
+			collaboratorMiddleName: value.middle_name,
+			collaboratorLastName: value.last_name,
+			collaboratorInstitutionalEmail: value.institutional_email,
+			collaboratorPositionTitle: value.position_title,
+			collaboratorPrimaryAffiliation: value.title,
+			collaboratorResearcherProfileURL: value.profile_url,
+			collaboratorSuffix: value.suffix,
+			collaboratorType: value.collaborator_type,
```

Comment:
The issue here is the keys are actually different, the DTO uses `collaboratorFirstName` vs the db which is simply `first_name`